client/scripts/containers/Room/RoomContainer.js

```diff
@@ -50,7 +65,8 @@ const Room = compose(
   updateToken,
   deleteMessage,
   addReaction,
-  roomQuery
+  roomQuery,
+  connect(mapStateToProps, mapDispatchToProps)
```

Comment:
I moved the `connect()` down so that the Apollo data props would be accessible to `mapStateToProps`.